packages/calling-stateful-client/src/Converter.ts

```diff
@@ -27,9 +26,9 @@ import {
 } from './CallClientState';
 import { CaptionsInfo } from './CallClientState';
 
-/* @conditional-compile-remove(teams-identity-support) */ /* @conditional-compile-remove(meeting-id) */
+/* @conditional-compile-remove(teams-identity-support) */
```

Comment:
For ones with multiple conditional-compiles, remove all comments (they work as or statements, if any of them are stable then they are included, so once one of them permanently stabilizes the line should always exist)

Reply:
hi @JamesBurnside this import are not used for meeting-id and tags was added in development. Currently this import still used by "teams-identity-support" and will throw compilation error if this tag will be removed.

Reply:
Teams Identity support is also in GA so removing it should be fine 😊